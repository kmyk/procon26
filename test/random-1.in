11111100000000000000111111111111
00100100011000011000011111111111
00111011011111101100011111111111
01011011111000001000011111111111
01000100000000000000011111111111
01001000000000001111101111111111
11001101101000001100101111111111
10010000110100001001011111111111
11100001000100111010101111111111
01100001000011100100101111111111
01111110110001011100101111111111
01100110111110111111001111111111
01101001000001000000001111111111
11010001111001000000001111111111
11101101000100000000101111111111
10100001000101111100101111111111
10100001000101111110101111111111
10101110100101000110101111111111
00010010100111111110111111111111
00010010101000111001101111111111
00001101101111111111101111111111
11101111111111000000001111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111

17
00011100
00000100
00000100
00000100
00000100
00100110
00111110
00000000

00000000
00000000
00011110
00011110
00011111
00011000
00000000
00000000

00000000
00001000
00001000
00001000
00111000
00111000
00111000
00000000

00000000
00000000
00000000
00000000
00000000
01110000
01110000
00111000

00000000
00000010
00001111
00001101
00001100
00001100
00001100
00000000

00000000
00000001
00000001
00000111
00000110
00000110
00000110
00000000

00000000
01110000
01110000
01111000
00001000
00011000
00000000
00000000

00000000
11110000
11110000
11110000
00011000
00011000
00000000
00000000

00000110
00000110
00001110
00001100
00001110
00000000
00000000
00000000

00000000
00001110
00011110
00001110
00000110
00000110
00000000
00000000

00011100
00010100
00010100
00010100
00000100
00011100
00000000
00000000

00000000
00000000
00000000
00000100
01111100
00001100
00001100
00000000

00000000
00000000
00000000
00000000
01000000
01111111
01111111
00000000

00000000
11111100
11111100
00011100
00000000
00000000
00000000
00000000

00000000
00000000
00000000
00000110
00000100
00000110
00000110
00000000

11111110
01110000
00111110
00000000
00000000
00000000
00000000
00000000

00000000
00000000
00000110
00011110
00001000
00000000
00000000
00000000
