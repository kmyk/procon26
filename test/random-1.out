

14 0 T 270





12 3 T 0



6 3 H 180



3 3 H 180
